File: demo/demo_1/solution.go
package problem1

import (
	"encoding/json"
	"log"
	"strings"
)

func twoSum(nums []int, target int) []int {

}

func Solve(input string) interface{} {
	values := strings.Split(input, "\n")
	var nums []int
	var target int

	if err := json.Unmarshal([]byte(values[0]), &nums); err != nil {
		log.Fatal(err)
	}
	if err := json.Unmarshal([]byte(values[1]), &target); err != nil {
		log.Fatal(err)
	}

	return twoSum(nums, target)
}

File: demo/demo_1/Solution.java
package demo.demo_1;

import com.alibaba.fastjson.JSON;
import java.util.*;
import qubhjava.BaseSolution;


public class Solution extends BaseSolution {
    public int[] twoSum(int[] nums, int target) {

    }

    @Override
    public Object solve(String[] values) {
        int[] nums = jsonArrayToIntArray(values[0]);
		int target = Integer.parseInt(values[1]);
        return JSON.toJSON(twoSum(nums, target));
    }
}
